sources.cfg

```diff
@@ -0,0 +1,6 @@
+[buildout]
+extends =
+    http://kgs.4teamwork.ch/sources.cfg
+    https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/4teamwork-psc.cfg
```

Comment:
✂️ This includes private sources. But since this package is public, it should not have private dependencies, and thus this line should not be required.